meilisearch-release-changelog/v1.11.0.md

```diff
@@ -0,0 +1,208 @@
+
+```json5
+POST /multi-search
+{
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	"facetsByIndex": {
+	  "movies": ["title", "id"],
+	  "comics": ["title"],
+	}
+  },
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+### Request merged facet distribution and stats
+
+Merge the returned facets in a single facet distribution and stats that is global to the entire request, by using `federation.mergeFacets` in the POST `POST /multi-search` route.
+
+The number of returned values can be limited for all facets with `federation.mergeFacets.maxValuesPerFacet`.
+
+```json5
+POST /multi-search
+{
+
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	  "facetsByIndex": {
+	    "movies": ["title", "id"],
+	    "comics": ["title"],
+	  },
+	  "mergeFacets": {
+	    "maxValuesPerFacet": 10,
+	  }
+  }
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+Done by @dureuill in #4929.
+
+## Experimental - `STARTS_WITH` operator during search
+
+Enable the experimental feature to use the `STARTS_WITH` filter operator:
+
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "containsFilter": true
+  }'
+```
+
+Use the `STARTS_WITH` operator when filtering:
+
+```json5
+curl \
+  -X POST http://localhost:7700/indexes/movies/search \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "filter": "hero STARTS_WITH spider"
+  }'
+```
+
+🗣️ This is an experimental feature, and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/763).
+
+Done by @Kerollmops in #4939.
+
+## Other improvements
+
+* Language support and [localizedAttributes settings](https://www.meilisearch.com/docs/reference/api/settings#localized-attributes) by @ManyTheFish in #4937
+  * Add iso-639-1 variants
+  * Convert iso-639-1 into iso-639-3
+
+# Fixes 🐞
+
+* ⚠️ When using federated search, `query.facets` was silently ignored at the query level, but should not have been. It now returns the appropriate error. Use `federation.facetsByIndex` instead if you want facets to be applied during federated search.
+* In prometheus metrics return the route pattern instead of the real route when returning the HTTP requests total by @irevoire in #4839
+* Always truncate values at the end of the list of facet values when the number of facet values is larger than the index setting `faceting.maxValuesPerFacet`. For example, for the facet values ordered alphabetically `["blue", "red", "yellow"]`, setting `faceting.maxValuesPerFacet` sometimes resulted in `["blue", "yellow"]` instead of ["blue", "red"]`. By @dureuill in #4929
```

Comment:
```suggestion
* Truncate values at the end of a list of facet values when the number of facet values is larger than `maxValuesPerFacet`. For example, setting `maxValuesPerFacet` to `2` could result in `["blue", "red", "yellow"]`, being truncated to `["blue", "yellow"]` instead of ["blue", "red"]`. By @dureuill in #4929
```